db/migrate/20150828024032_devise_create_users.rb

```diff
@@ -13,11 +15,11 @@ def change
       t.datetime :remember_created_at
 
       ## Trackable
-      t.integer  :sign_in_count, default: 0, null: false
-      t.datetime :current_sign_in_at
-      t.datetime :last_sign_in_at
-      t.string   :current_sign_in_ip
-      t.string   :last_sign_in_ip
+      # t.integer  :sign_in_count, default: 0, null: false
```

Comment:
Is this the kind of repo we want to be adjusting the migrations instead of creating new ones? Just double checking

Reply:
It's not in production anywhere... And the upgrade is probably having some breaking changes so it should be ok :)